examples/step-17/step-17.cc

```diff
@@ -282,58 +290,97 @@ namespace Step17
   // @sect4{ElasticProblem::setup_system}
 
   // Next, the function in which we set up the various variables
-  // for the global linear system to be solved is implemented.
+  // for the global linear system to be solved needs to be implemented.
+  //
+  // However, before we with this, there is one thing to do for a
+  // parallel program: we need to determine which MPI process is
+  // responsible for each of the cells. Splitting cells among
+  // processes, commonly called "partitioning the mesh", is done by
+  // assigning a @ref GlossSubdomainId "subdomain id" to each cell. We
+  // do so by calling into the METIS library that does this in a very
+  // efficient way, trying to minimize the number of nodes on the
+  // interfaces between subdomains. Rather than trying to call METIS
+  // directly, we do this by calling the
+  // GridTools::partition_triangulation() function that does this at a
+  // much higher level of programming.
+  //
+  // @note As mentioned in the introduction, we can avoid this manual
```

Comment:
I think 'can' should be replaced by 'could' (because it fits the 'used' in the next line)